protocols/v2/const-sv2/README.md

```diff
@@ -14,8 +16,7 @@
 
 ## Usage
 
-To use this crate, add it to your `Cargo.toml`:
+To include this crate in your project, run:
 
-```toml
-[dependencies]
-const_sv2 = "2.0.0"
+```bash
```

Comment:
Can both `bash` and `sh` be used interchangeably here? Does using `sh` change the syntax coloring? If there is not difference, leave as `bash`.

```suggestion
```sh
```